File: Simplicity/color.cpp
#include "color.h"

Color::Color(float r, float g, float b) {
	this->r = r;
	this->g = g;
	this->b = b;
	this->w = 255;
}

Color::Color() : Color(255, 255, 255) {

}

Color::Color(float r, float g, float b, float w) : Color(r,g,b) {
	this->w = w;
}

File: Simplicity.NET/Simplicity.NET/Graphics/Objects/Color.cs
namespace Simplicity.NET.Graphics.Objects
{
    public struct Color
    {
        public float R;
        public float G;
        public float B;
        public float W;

        public Color(float r, float g, float b)
        {
            R = r;
            G = g;
            B = b;
            W = 255;
        }

        public Color(float r, float g, float b, float alpha)
        {
            R = r;
            G = g;
            B = b;
            W = alpha;
        }
    }
}
